lib/create-response-handler.ts

```diff
@@ -17,9 +17,11 @@ import { TranslatedMatchingResponseBody, TranslatedIdentityResponseBody, Scenari
  * ResponseScenarios
  *
  * An object enumerating scenarios that can occur when handling a response from
```

Comment:
```suggestion
 * An object listing scenarios that can occur when handling a response from
```
`enumerating` feels long and cumbersome.